### ISSUE TITLE
Add to Homescreen Contextual Hint

### ISSUE THREAD

**Issue by Sdaswani** [mozilla-mobile/focus-android#2897]
Per #2816 (comment) @brampitoyo should think of a nice contextual hint to ask users to add stuff to homescreen.

**Comment by brampitoyo** [comment]
![add-to-homescreen 2x](https://user-images.githubusercontent.com/51007/43119845-2d149606-8f6c-11e8-8027-caab8ba467e6.png)

**Comment by Sdaswani** [comment]
@brampitoyo do I understand that this shows only when the user clicks into the menu?

**Comment by brampitoyo** [comment]
That’s right. The thinking goes that the user knows to tap the menu whenever they want to add a bookmark (as is the case with both Chrome and Firefox for Android). So, when they want to do that on Focus, they would do the same and see the hint to add a homescreen shortcut – at the right place and moment.

**Comment by Sdaswani** [comment]
Out of 400 million app foregrounds, the menu has only been clicked some 9 million times - that's about a 2% chance users will ever see this tip.

**Comment by kfontecilla** [comment]
@brampitoyo Could we get an update on this issue? See susheel's comment above. Thx

**Comment by brampitoyo** [comment]
@kfontecilla While labelling the menu is still a good idea, we have now chosen to have “Add to homescreen” be a tip on the start screen (#2816).

I think that Susheel’s comment (that the menu isn’t clicked often) will be addressed when we expose a specific start screen tip to users who have never accessed this feature.

**Comment by bbinto** [comment]
@brampitoyo is this ready for eng-breakdown?

**Comment by brampitoyo** [comment]
@bbinto Yes. This is ready for engineering breakdown.

The new description should be:

> **Add to home screen**
> Access this site in one-tap

**Comment by boek** [comment]
@bbinto should we move this out of 7.0 and into the backlog? These will require more strings

**Comment by Sdaswani** [comment]
This does support our 'Homescreen' OKR but I think we have that covered with home screen tips so I think it should be moved out too.

**Comment by bbinto** [comment]
Fine by me, I didn't expect any just in time features to land in 7.0 anyway, this was mostly the UX work that had to be done.

**Comment by boek** [comment]
Just removing the milestone so we can triage this in the next meeting :)
